nampower/main.cpp

```diff
@@ -249,7 +251,8 @@ void SignalEventHook(hadesmem::PatchDetourBase *detour, game::Events eventId)
     // the server perceives too little time as having passed to allow another cast.  i dont
     // think there is anything we can do about this except to honor the servers request to
     // abort the cast.  reset our cooldown and allow 
-    else if (eventId == game::Events::SPELLCAST_FAILED && !gNotifyServer)
+    else if (eventId == game::Events::SPELLCAST_FAILED/* && !gNotifyServer*/ || //JT: !gNotifyServer breaks QuickHeal
+            eventId == game::Events::SPELLCAST_INTERRUPTED) //JT: moving to cancel the spell sends "SPELLCAST_INTERRUPTED"
         gCooldown = 0;
```

Comment:
What is QuickHeal?  From what I can recall, the value of `gNotifyServer` is only false when the server is responsible for the event we are processing.

Reply:
https://github.com/codefracas/QuickHeal

Quickheal did not work at all w/o this change.